aidants_connect_web/templates/public_website/faq/mandat.html

```diff
@@ -207,6 +204,13 @@ <h3 id="Comment-l’usager-signe-t-il-le-mandat-">
           <li>Une fois le mandat créé, la structure imprime le mandat en deux exemplaires&nbsp;: un premier qu'elle remet à l'usager, un second signé par l'usager qu'elle conserve et archive.</li>
           <li>Après chaque validation, Aidant Connect crée une empreinte numérique unique (appelée «&nbsp;Hash&nbsp;») qui permet de garantir l’intégrité du mandat, c’est-à-dire le fait que le document n’a subi aucune modification (volontaire ou involontaire).</li>
         </ol>
+        <h3 id="Pendant combien de temps conserver les mandats papiers">
```

Comment:
On évite les espaces dans les id en HTML. Donc plutôt des tirets ou des underscore : 
```suggestion
        <h3 id="conserver_mandats_papier">
```